javascript/javascript1/week2/flight-booking.js

```diff
@@ -0,0 +1,9 @@
+function getfullname(firstname, lastname, useformal){
+  fullName = firstname + '' + lastname;
+  if (useformal) {
```

Comment:
Good 👍  Try to use ES6 specifications like "Template literals"(backticks) for concatenation and Terinary operators.